.github/workflows/generate-header-demo.yaml

```
@@ -129,100 +129,140 @@ jobs:
           path: docs/img/*.gif
           retention-days: 30
 
-      - name: Upload demo to orphan branch and prepare preview
+      - name: Upload all demos to orphan branch and prepare preview
         if: steps.context.outputs.is_pr == 'true'
         id: demo_preview
         run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)